arelle/ViewFileRenderedGrid.py

```diff
@@ -730,7 +730,7 @@ def yAxisByCol(self, leftCol, row, yParentStrctNode, renderNow, atTop):
                 else:
                     row = nextRow
                 if nestRow > nestedBottomRow:
-                    nestedBottomRow = nestRow + (isNonAbstract and not childrenFirst)
+                    nestedBottomRow = nestRow + isNonAbstract # and not childrenFirst)
```

Comment:
Not quite certain of this change, I don't know if this variable is still needed or not, it's not initialized and the code that used it before (below)  is in a comment